server/routes/event.js

```diff
@@ -0,0 +1,11 @@
+const EventController = require('../controllers/eventController')
+const { authorize } = require('../middleware/auth')
+
+const router = require('express').Router()
+
+router.get('/',EventController.showAllEvent)
+router.post('/',EventController.addEvent)
+router.delete('/:id',authorize,EventController.deleteEvent)
```

Comment:
bisa dipasang gini juga yah 
```suggestion
router.use(':/id', authorize)
router.delete('/:id',EventController.deleteEvent)
router.put('/:id',EventController.editEvent)
```